docs/en/observability/apm/collect-application-data/index.asciidoc

```diff
@@ -52,15 +50,15 @@ Are we ok with the gaps it exposes?
 [cols="<,^,^"]
 |===
 | *Language*  | *Elastic APM agent* | *Elastic Distributions of OpenTelemetry (EDOT)*
-| *Android*   | Android agent | {not-available}
-| *Go*        | Go agent      | {not-available}
-| *iOS*       | iOS agent     | {not-available}
-| *Java*      | Java agent    | EDOT Java
-| *.NET*      | .NET agent    | preview:[] EDOT .NET
-| *Node.js*   | Node.js agent | preview:[] EDOT Node.js
-| *PHP*       | PHP agent     | preview:[] EDOT PHP
-| *Python*    | Python agent  | preview:[] EDOT Python
-| *Ruby*      | Ruby agent    | {not-available}
+| *Android*   | {not-available} | Elastic OTel Android Agent
+| *Go*        | Go agent        | {not-available}
+| *iOS*       | {not-available} | Elastic APM iOS Agent
+| *Java*      | Java agent      | EDOT Java
+| *.NET*      | .NET agent      | EDOT .NET
+| *Node.js*   | Node.js agent   | EDOT Node.js
+| *PHP*       | PHP agent       | EDOT PHP
+| *Python*    | Python agent    | EDOT Python
+| *Ruby*      | Ruby agent      | {not-available}
```

Comment:
As mentioned in https://github.com/elastic/docs-content/pull/1099, I'm not sure what to do about Android and iOS. They are listed in the Elastic OpenTelemetry docs as EDOT SDKs, but those links go back to the APM agent docs where they are called Elastic OTel Android Agent and Elastic APM iOS Agent (not EDOT SDKs). Any guidance would be appreciated!

Reply:
Those distros currently lack documentation. I'd say let's link to the GH repos for now.